src/dmtcpplugin.cpp

```diff
@@ -130,6 +131,15 @@ EXTERNC int dmtcp_checkpoint()
   return rv;
 }
 
+EXTERNC int dmtcp_checkpoint_ckptDir(const char* ckptDir)
+{
+  if (ckptDir != NULL) {
+    CoordinatorAPI::instance().updateCkptDirGlobally(ckptDir);
+  }
+  int rv = dmtcp_checkpoint();
```

Comment:
I think that's better idea we can simply create API to set checkpoint directory globally for all processes and then user can make explicit call to dmtcp_checkpoint whenever required.  I created single call for Veloce Integration but now it sounds logical to create one API which sets ckpt directory globally